### PR TITLE
E2E for sidebar and link navigation of system admin console

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
E2E for sidebar and link navigation of system admin console

Test steps:
1. Visit the admin console directly via URL and verify that it matches content's header
2. Go to other admin URL, then click the section on the admin sidebar.  Verify that it matches URL and content's header

Note:
- Specs are separated for those in enterprise and non-enterprise features.
- Pre-packaged plugins will be tested separately

#### Ticket Link
none

### CODE REVIEW

File: e2e/cypress/integration/enterprise/system_console/sidebar_link_navigation_spec.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    ];
+
+    testCases.forEach((testCase) => {
+        it(`can navigate to ${testCase.header}`, () => {
+            // # Visit the URL directly
+            cy.visit(testCase.url);
+
+            // * Verify that the header is correct
+            cy.get('.admin-console').should('be.visible').within(() => {
+                cy.get('.admin-console__header').should('be.visible').and(testCase.headerContains ? 'contain' : 'have.text', testCase.header);
```
[the remark this answers is not in the document]
Comment (saturninoabril) [reply e2e/cypress/integration/enterprise/system_console/sidebar_link_navigation_spec.js]: Done

File: e2e/cypress/integration/system_console/sidebar_link_navigation_spec.js [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        {
+            header: 'Experimental Features',
+            sidebar: 'Features',
+            url: 'admin_console/experimental/features',
+            otherUrl: 'admin_console/about/license',
+        },
+    ];
+
+    testCases.forEach((testCase) => {
+        it(`can navigate to ${testCase.header}`, () => {
```
[the remark this answers is not in the document]
Comment (saturninoabril) [reply e2e/cypress/integration/system_console/sidebar_link_navigation_spec.js]: Okay, in the describe, I changed to `'System Console - Enterprise'`

File: e2e/cypress/integration/system_console/sidebar_link_navigation_spec.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+    });
+
+    it('can go to admin console by clicking System Console', () => {
+        cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
+
+        cy.get('.Menu__content').should('be.visible').within((container) => {
+            cy.getByText(/System Console/, {container}).should('be.visible').click();
+        });
+
+        cy.url().should('include', '/admin_console/about/license');
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/sidebar_link_navigation_spec.js]: @prapti I couldn't reproduced the one you're seeing where above test failed.  "System Console" has href of `/admin_console` which redirects to `/admin_console/about/license` on click.  How about let's keep the above check as is and see if the same will fail on daily test?

Comment (prapti) [reply e2e/cypress/integration/system_console/sidebar_link_navigation_spec.js]: Sounds good @saturninoabril. Thank you for following up on the error!

**Comment by saturninoabril** [comment]
@srkgupta @bradjcoughlin Updated per latest change on Cypress upgrade.  Ready for your review, thanks

**Pull request merged**